### DISCUSSION TITLE
Adult Video Database v0.8.2-beta (Carter)

### DISCUSSION THREAD

**Discussion by fupdec** [fupdec/Adult-Video-Database#77] [General]
<b>Subsribe on <a href="https://www.patreon.com/avdb">Patreon</a> to get license key </b>

## Added
- app registration with license key
- settings for checking updates
## Fixed
- items selection #64

<hr /><em>This discussion was created from the release <a href='https://github.com/fupdec/Adult-Video-Database/releases/tag/v0.8.2-beta'>Adult Video Database v0.8.2-beta (Carter)</a>.</em>

**Comment by ThePlexSwede** [comment]
Hi there,

Just found you application yesterday and I have to say I am very impressed by what I have seen so far.

I had a look at the YouTube video showing the functions etc. Which covers pretty much everything I am looking for apart from one thing. Is it possible to scrape the movie information of your Adult DVDs from IAFD, similar to how it’s done for the performers? (Btw. How this is implemented for the performers is very neat)

Is this something that is available but not showing in the video or planned? 

Thanks

D

**Reply by fupdec** [reply]
Not yet available for video. Will be available when support for Movies is added (several videos combined into one card with their own cover, description, etc.). Perhaps in a couple of months.
